src/SIM/AdaptiveSIM.h

```diff
@@ -115,7 +115,7 @@ class AdaptiveSIM : public SIMadmin
   int nBlock; //!< Running VTF result block counter
 
   std::vector<Vector>      projs;  //!< Projected secondary solutions
-  std::vector<const char*> prefix; //!< Norm prefices for VTF-output
+  std::vector<std::string> prefix; //!< Norm prefices for VTF-output
```

Comment:
Is there a good reason (memory leak, etc.) for this change? I assume yes, or is it just a convenience matter.

Reply:
It used to point to local variables, so  I changed it to get copies since they now live in a larger scope